content/v2.0/influxdb-templates/stacks/init.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Initialize an InfluxDB stack
+list_title: Initialize a stack
+description: >
+  InfluxDB automatically creates a new stack each time you [apply an InfluxDB template](/v2.0/influxdb-templates/use/)
+  **without providing a stack ID**.
+  You can also use the [`influx stacks init` command](/v2.0/reference/cli/influx/stacks/init/)
+  to manually create or initialize a new stack.
+menu:
+  v2_0:
+    parent: InfluxDB stacks
+    name: Initialize a stack
+weight: 201
+related:
+  - /v2.0/reference/cli/influx/stacks/init/
+list_code_example: |
+  ```sh
+  influx apply \
+    -o example-org \
+    -f path/to/template.yml
+  ```
+  ```sh
+  influx stacks init \
+    -o example-org \
+    -n "Example Stack" \
+    -d "InfluxDB stack for monitoring some awesome stuff" \
+    -u https://example.com/template-1.yml \
+    -u https://example.com/template-2.yml
+  ```
+---
+
+InfluxDB automatically creates a new stack each time you [apply an InfluxDB template](/v2.0/influxdb-templates/use/)
+**without providing a stack ID**.
+You can also use the [`influx stacks init` command](/v2.0/reference/cli/influx/stacks/init/)
+to manually create or initialize a new stack.
+
+## Initialize a stack when applying a template
+To automatically create a new stack when [applying an InfluxDB template](/v2.0/influxdb-templates/use/)
+**don't provide a stack ID**.
+InfluxDB applies the templated resources to a new stack and provides the **stack ID** the output.
+
+```sh
+influx apply \
+  -o example-org \
+  -f path/to/template.yml
+```
+
+## Manually initialize a new stack
+Use the [`influx stacks init` command](/v2.0/reference/cli/influx/stacks/init/)
+to create or initialize a new InfluxDB Stack.
+
+**Provide the following:**
+
+- Organization name or ID
+- Stack name
+- Stack description
+- InfluxDB template URLs
+
+<!-- -->
```

Comment:
line 59 intentional?

Reply:
Yes. The markdown parser doesn't know when the list above is supposed to end so it will actually try to nest the code block in the list. The comment forces the markdown parser to close the list and render the code block as a separate element. The comment line doesn't show up on the page and the list and code block render correctly.